src/colors/color.rs

```diff
@@ -337,6 +337,48 @@ impl Color {
         (long, medium, short)
     }
 
+    /// convert normalized RGB value to linear value
+    fn to_linear(u: f32) -> f32 {
+        if u >= 0.04045 {
+            ((u + 0.055) / (1. + 0.055)).powf(2.4)
+        } else {
+            u / 12.92
+        }
+    }
+
+    /// Convert color to OKLAB color space
+    ///
+    /// https://bottosson.github.io/posts/oklab/
```

Comment:
```suggestion
    /// Converts the color to OKLAB color space.
    ///
    /// For more information see <https://bottosson.github.io/posts/oklab/>.
```
This was rust doc will render it as a clickable link.